compose/ChartboostMediationDemo/build.gradle.kts

```diff
@@ -54,7 +54,6 @@ dependencies {
     implementation("com.chartboost:chartboost-mediation-sdk:4.7.1")
 
     // Chartboost Mediation adapters
-    implementation("com.chartboost:chartboost-mediation-adapter-adcolony:4.+")
```

Comment:
Anything in the README.md?

Reply:
Just in the CHANGELOG and we don't want to touch that.